### DISCUSSION TITLE
Adding the ability to await a Tick Using Coroutines and Async/Await

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#532] [Feature Request]
Looking for 2 built in ways to wait for a tick. 

One Using Coroutines,

yield return new WaitForTick();

One using AsycAwait

await Task.Run(() => TimeManager.WaitForTick());

There may be a good reason as these do not exist that I do not know of, but just asking :)

I know I can build my own using TimeManager.OnTick() and have made them, but it would be nice if Fishnet had them build in natively.

**Comment by FirstGearGames** [comment]
This might be possible. Have not looked into it yet.
